simpa/core/simulation_modules/reconstruction_module/__init__.py

```diff
@@ -54,13 +54,14 @@ def run(self, device):
             raise TypeError(f"Type {type(device)} is not supported for performing image reconstruction.")
 
         if Tags.RECONSTRUCTION_PERFORM_BANDPASS_FILTERING in self.component_settings and \
-                self.component_settings[Tags.RECONSTRUCTION_PERFORM_BANDPASS_FILTERING] is True:
+                self.component_settings[Tags.RECONSTRUCTION_PERFORM_BANDPASS_FILTERING] and \
+                    Tags.BANDPASS_FILTER_METHOD in self.component_settings:
```

Comment:
This means that if I request bandpass filtering and don't specify a filter method as a user, no filtering will be applied.
Should we rather define a default method and only change it if requested?